content/intermediate/operations/understanding-changesets/index.md

```diff
@@ -39,9 +67,81 @@ Let’s get started!
     8. In the review page, choose **Create Stack**.
     9. Refresh the stack creation page until you see your stack in the `CREATE_COMPLETE` status.
 
+   ::::
+   :::::
 ### Lab part 1
 In this part of the lab, you will specify a property, for a given resource type, that requires [no interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt) on stack updates. You will then create a change set to preview the changes, and inspect the output of the change set operation.
 
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
+Open the `bucket.yaml` CloudFormation template in your `Cloud9` editor, and add `VersioningConfiguration` as shown below. Save the file.
```

Comment:
Suggest moving the open/edit of template outside the tabbed instructions

Reply:
the instructions were specific to Cloud9 and it is different for local development.